ome-dundeeomero.yml

```diff
@@ -245,29 +256,38 @@
     #omero_server_db_dumpdir_parent: /tmp
     #omero_server_db_dumpdir_name: nightly-pg_dump_omero.dir
 
-    ice_version: "3.6"
+    nginx_version: 1.16.1
```

Comment:
Should I maybe bump demo up to this from its current 1.14.2?

Reply:
There's no harm in updating demo, but I don't think there's a strong need to at the moment either.